3.9-minimal/test/run

```diff
@@ -199,7 +199,12 @@ test_application() {
   # that the container starts at all
   check_result $?
 
-  test_scl_usage "python --version" "Python $VERSION." "${cid_file}"
+  # Instead of relying on VERSION variable coming from Makefile
+  # set the expected string based on the PYTHON_VERSION defined
+  # inside the running container.
+  python_version=$(docker run --rm $IMAGE_NAME /bin/bash -c "echo \$PYTHON_VERSION" 2>&1)
```

Comment:
Above in this script there is `if [[ -z $VERSION ]]; ... exit 1`, so why can't we rely on `VERSION`?

In addition, if there is a reason to detect the `python_version` like this, wouldn't it be better to do it outside of `test_application` so it runs only once?

Reply:
$VERSION works only if it is 3.7 or 3.8 but not for 3.9-minimal because then the $VERSION is not a part of `python -V` output. That's the main reason. Now the test also tests that the env variable is correct in the container so I don't think it's bad.

Reply:
Ok, sounds good.